core/pom.xml

```diff
@@ -362,6 +362,24 @@
       </extension>
     </extensions>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <annotationProcessorPaths>
```

Comment:
I just noticed a `run-annotation-processor` profile that adds `error_prone_docgen_processor` to the dependencies, and would need to be changed to add to the `annotationProcessorPaths` instead (using `combine.children="append"`, see https://maven.apache.org/pom.html#plugins)

----

(docgen_processor is also added as a dependency to docgen, but AFAICT this is for its `BugPatternInstance` class, not for its annotation processor, so no need to add it to the processor path there)

Reply:
As a first pass I ended up just declaring the whole block. I worry that the `combine.children` might later confuse people down the line. If you have a preference for the `combine.children` approach I'm happy to make the change.